x-pack/test/functional/page_objects/gis_page.js

```diff
@@ -92,7 +92,10 @@ export function GisPageProvider({ getService, getPageObjects }) {
     async waitForLayerDeleted(layerName) {
       log.debug('Wait for layer deleted');
       await retry.try(async () => {
```

Comment:
Lets reimplement this as a `retry.waitFor()` call, so it's a little less noisy in the logs and doesn't look like something failing.

```js
await retry.waitFor('Layer to be deleted', async () => await this.doesLayerExist(layerName))
```